Update Wazuh docs (#1058)
stackhpc/stackhpc-kayobe-config: doc/source/configuration/wazuh.rst

  kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/wazuh-secrets.yml
  ansible-vault encrypt --vault-password-file ~/vault.pass $KAYOBE_CONFIG_PATH/wazuh-secrets.yml

Configure Wazuh Dashboard's Server Host 
---------------------------------------

> It is best practice to manually set the dashboard's server host.

Reviewer: ```suggestion
It is common to want to configure the Wazuh dashboard to serve on a different network than the overcloud provisioning network used for Wazuh's internal communication.
```

Author: Applied.

> By default, the Dashboard will be hosted on a random IP. It is best practice to manually set the dashboard's server host.

Reviewer: As discussed, I don't think this is true. You can follow the variables in SKC:

```
dashboard_server_host: "{{ indexer_network_host }}"
indexer_network_host: "{{ provision_oc_net_name | net_ip }}"
```

So it uses the overcloud provisioning network, like other Wazuh communication.

Author: Has been amended and changed.

> By default, the Dashboard will be hosted on a random IP. It is best practice to manually set the dashboard's server host.

Reviewer 2: Very nitpicky, but I don't like the phrase "a random IP". Can we just drop this sentence?
```suggestion
It is best practice to manually set the dashboard's server host.
```

> By default, the Dashboard will be assigned the ``0.0.0.0`` server host address, meaning it will likely inheret the server host IP address from the last network interface to be added to ``infra-vm`` within ``$KAYOBE_CONFIG_PATH/environments/<env_name>/inventory/group_vars/wazuh-manager/network-interfaces.yml``. Due to this uncertainty, as well as the different networks likely having different firewall rulesets which may interfere with the Wazuh Dashboard service, it is best practice to manually set the dashboard's server host.

Reviewer 3: Making docs more verbose makes them less clear
Fewer words is better
```suggestion
By default, the Dashboard will be hosted on a random IP. It is best practice to manually set the dashboard's server host.
```

In order to do so, either create or edit the ``$KAYOBE_CONFIG_PATH/environments/<env_name>/inventory/group_vars/wazuh-manager/wazuh-manager.yml`` configuration file to include the dashboard variable:

.. code-block:: yaml
   :caption: $KAYOBE_CONFIG_PATH/environments/<env_name>/inventory/group_vars/wazuh-manager/wazuh-manager.yml

    dashboard_server_host: "{{ <network-name-prefix>_net_name | net_ip }}"

> Where ``<network-name-prefix>`` refers to the unique name attached to the front of the various network roles, often defined as a list of ``net_name``s at the top of the Kayobe network configuration file, ``$KAYOBE_CONFIG_PATH/environments/<env_name>/networks.yml``. For example, to assign the Wazuh dashboard the server host attatched to the ``external_intranet`` network, one may find within ``networks.yml`` that ``public_net_name: external_intranet`` is set, resulting in the following Wazuh dashboard configuration:

Reviewer 3: It can be assumed that the consumers of the docs will have a basic understanding of kayobe networks. In any case, this isn't the place to explain it. 
```suggestion
For example:
```

Reviewer 3: @MaxBed4d I think you missed this

.. code-block:: yaml
   :caption: $KAYOBE_CONFIG_PATH/environments/<env_name>/inventory/group_vars/wazuh-manager/wazuh-manager.yml

    dashboard_server_host: "{{ public_net_name | net_ip }}"

If this is being added post deployment the user will be required to re-run the ``wazuh-manager.yml`` ansible playbook via the following command:

> .. code-block:: yaml

Reviewer 3: ```suggestion
.. code-block:: bash
```

   :caption: Deploy or re-run the ``wazuh-manager.yml`` ansible playbook to apply changes made to the configuration.

    kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/wazuh-manager.yml

> More on the deployment of Wazuh dashboard can be found below in the :ref:`subsequent section <Deploy>`.

Reviewer 3: You can use an implicit reference here https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#implicit-targets
